Lib/test/test_tokenize.py

```diff
@@ -39,20 +39,22 @@ def test_basic(self):
     """)
         self.check_tokenize("if False:\n"
                             "    # NL\n"
+                            "    \n"
                             "    True = False # NEWLINE\n", """\
     NAME       'if'          (1, 0) (1, 2)
     NAME       'False'       (1, 3) (1, 8)
     OP         ':'           (1, 8) (1, 9)
     NEWLINE    '\\n'          (1, 9) (1, 10)
     COMMENT    '# NL'        (2, 4) (2, 8)
     NL         '\\n'          (2, 8) (2, 9)
-    INDENT     '    '        (3, 0) (3, 4)
-    NAME       'True'        (3, 4) (3, 8)
-    OP         '='           (3, 9) (3, 10)
-    NAME       'False'       (3, 11) (3, 16)
-    COMMENT    '# NEWLINE'   (3, 17) (3, 26)
-    NEWLINE    '\\n'          (3, 26) (3, 27)
-    DEDENT     ''            (4, 0) (4, 0)
+    NL         '\\n'          (3, 4) (3, 5)
```

Comment:
Because there are two lines between the comment an "True". One ends the comment, and other ends the blank line (spaces are ignored).

Look closer on line numbers. "if" is at line 1, the comment is at line 2, "True" is at line 4. Previously the tokenizer reported "True" at line 3.

The fifth line contains only DEDENT. 4 newlines divide the whole text on 5 parts. If you write the sample text in a file and open it with a text editor for sure you could move the cursor on the line 5, just after the 4th newline.

Reply:
Yes, I was confused by the DEDENT. Thank you for clarification!

Reply:
I was confused myself too. It looked to me that this change also fixed a bug in line numbering, but I was not able to find a bug in the old code. Later I figured out that the input data for this test was changed, therefore both old and new codes are correct. This PR just cleans up the code, as was claimed.